modules/imgimport/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-include_directories(include ../core/include)
+include_directories(include ../core/include ../core/src)
```

Comment:
Yes, except that target_link_libraries needs to be after add_library (since the ImageImport library doesn't exist until that line).